lib/link_preview/configuration.rb

```diff
@@ -77,5 +79,13 @@ def middleware
     def middleware=(*middleware)
       @middleware = middleware
     end
+
+    def default_content_aspect_ratio
+      @default_content_aspect_ratio ||= (3.0 / 4.0)
```

Comment:
Would it be better to have the default content aspect ratio be 16:9 since that's what most video cameras / DSLR's / phones shoot these days?